Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2366,7 +2369,8 @@ extension BrowserViewController: SearchViewControllerDelegate {
     }
     
     func searchViewControllerAllowFindInPage() -> Bool {
-        return tabManager.selectedTab?.webView?.url?.isAboutHomeURL == false
+        let url = tabManager.selectedTab?.webView?.url
+        return url?.isAboutHomeURL == false && url?.isInterstitialURL == false
```

Comment:
This `isInterstitialURL` is tricky because we now have one more state to track, easy to miss adding it in other places.
But I do not see any cool solution for this unfortunately